server/export.go

```diff
@@ -68,17 +69,25 @@ func ExportCmd(ctx *Context, cdc *codec.Codec, appExporter AppExporter) *cobra.C
 			doc.AppState = appState
 			doc.Validators = validators
 
+			doc.AppState = sdk.MustSortJSON(doc.AppState)
+
```

Comment:
Only appState has inconsistent objects, such as map, so only need to sort appState to make genesis.json more identical to the initial genesis.json.